release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3351,6 +3351,72 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-16-12-https-support-TransferProtocolTypes"]
+===== Supporting HTTPS for TransferProtocolTypes in Redfish APIs
+
+* TLS can be enabled for communication between ironic and the baseboard management controller (BMC) in the bootstrap phase of the install process by adding 'disableVirtualMediaTLS: false' to the Provisioning CR file created on disk by the installer. (link:https://issues.redhat.com/browse/OCPBUGS-39468[*OCPBUGS-39468*])
+
+[id="ocp-4-16-12-kubernetes-1-29-8"]
+===== Updating to Kubernetes version 1.29.8
+
+* This release contains the changes that come from the update to Kubernetes version 1.29.8. (link:https://issues.redhat.com/browse/OCPBUGS-39015[*OCPBUGS-39015*])
+
+[id="ocp-4-16-12-redirecting-git-advanced"]
+===== Redirecting on web console with Edit Source Code
+
+* There are 2 option in the _Git Advanced_ section of the web console: one option is to add a branch, tag, or commit ID and other option is to add the context directory. With this release, if you add the context directory of a particular branch, tag, or commit ID, you are redirected to that directory by selecting the *Edit source code* icon. If a branch, tag, or commit ID is not entered, you are redirected to the base url as previously expected. (link:https://issues.redhat.com/browse/OCPBUGS-38914[*OCPBUGS-38914*])
+
+[id="ocp-4-16-12-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when a large number of secrets in a cluster were fetched in a single call, the API timed out and the CCO threw an error and then restarted. With this release, the CCO pulls the secret list in smaller batches of 100 and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41234[*OCPBUGS-41234*])
+
+* Previously, Operator Lifecycle Manager (OLM) catalog source pods did not recover from node failure if the `registryPoll` field was `none`. With this release, OLM CatalogSource registry pods recover from cluster node failures and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41217[*OCPBUGS-41217*])
+
+* Previously, the Cluster ingress operator logged non-existent updates. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39324[*OCPBUGS-39324*])
+
+* Previously, the installation program failed to install an {product-title} cluster in the `eu-es` (Madrid, Spain) region on a {ibm-power-server-title} platform that was configured as an `e980` system type. With this release, the installation program no longer fails to install a cluster in this environment. (link:https://issues.redhat.com/browse/OCPBUGS-38502[*OCPBUGS-38502*])
+
+* Previously, the ingress controller `Degraded` status would not set because of an issue with the `CanaryRepetitiveFailures` condition transition time. With this release, the condition transition time is only updated when the condition status changes, instead of when the message or reason are the only changes. (link:https://issues.redhat.com/browse/OCPBUGS-39323[*OCPBUGS-39323*])
```

Comment:
```suggestion
* Previously, the Ingress Controller `Degraded` status would not set because of an issue with the `CanaryRepetitiveFailures` condition transition time. With this release, the condition transition time is only updated when the condition status changes, instead of when the message or reason are the only changes. (link:https://issues.redhat.com/browse/OCPBUGS-39323[*OCPBUGS-39323*])
```

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#ingress-controller